bigquery/setup.py

```diff
@@ -50,6 +50,7 @@
     ],
     "tqdm": ["tqdm >= 4.0.0, <5.0.0dev"],
     "fastparquet": ["fastparquet", "python-snappy"],
+    "test": ["psutil >= 5.6.7"],
```

Comment:
I don't know if we necessarily want test-dependencies in our `setup.py` files. I know it's a common practice, but in the cloud clients I've seen test deps added to the `noxfile` as install steps in the session.

Reply:
Fair enough, will move the dependency then.